cmd/manager/main.go

```diff
@@ -84,8 +84,7 @@ func main() {
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
 		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
```

Comment:
This is:
- the webhook port (we don't have any webhooks)
- defaulted to `9443` anyway, even if we did have webhooks.